core-java-modules/core-java-exceptions/src/test/java/com/baeldung/exceptions/CheckedUncheckedExceptionsUnitTest.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.exceptions;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.FileNotFoundException;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests the {@link CheckedUncheckedExceptions}.
+ */
+public class CheckedUncheckedExceptionsUnitTest {
+
+    @Test
+    public void whenFileNotExist_thenThrowException() {
+        assertThrows(FileNotFoundException.class, () -> {
+            CheckedUncheckedExceptions.checkedExceptionWithThrows();
+        });
+    }
+
+    public void whenTryCatchExcetpion_thenSuccess() {
```

Comment:
Did you intend to add the Test annotation here?

Reply:
Added in the lastest push.